portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Comment{
```

Comment:
Small comment: Add a space here `Comment {` to make it easier to read :)

Reply:
Bigger comment (ha, this is too meta): Rename `Comment` to `Comments` or `CommentList`, since this data class is holding many comments